server/src/internalClusterTest/java/org/elasticsearch/indices/recovery/IndexRecoveryIT.java

```diff
@@ -808,27 +811,6 @@ public void testTargetThrottling() throws Exception {
 
         startShardRecovery(nodeA, nodeB);
 
-        logger.info("--> checking throttling increases on Node B (target node), while Node A (source node) reports no throttling");
```

Comment:
I've removed these. It seems the throttling stats is accumulative, so just asserting on the final state seems to be enough. I might be missing the actual intention behind these though!

Reply:
This is to ensure that we see some throttling before we relax the throttle. I think we want to stick with the tight throttle that we then relax rather than setting it looser to start with, or else this test might sometimes not do any throttling at all.

Reply:
(also note that we're not doing any relaxation here. just making sure target has a quite smaller threshold for throttling.)

Reply:
@pxsalehi can't we keep the original assertions? If the throttling setting is tighter I think that the assertions will hold?

Reply:
++ I'd rather keep the test pretty much as it is today and just relax the throttle on the node that we don't expect to do any throttling.

Reply:
TBH, I do not entirely agree. The point of the change was to simplify the test so we don't need that unthrottling and the assertions in the middle. We also could still check with a wide range of setting values rather than limiting the test to just the max (200mb). But in the interest of closing the test failure, I'm fine with just using 200mb to address the failure. See https://github.com/elastic/elasticsearch/pull/103397/commits/828e10b2146308b838117742b62aff3dcee608e3.